plugin/Field/Collection/FieldsCollectionToContentFieldLabelArrayConverter.php

```diff
@@ -45,6 +47,9 @@ public function convert(FieldsCollection $pFieldsCollection, string $module): ar
 		$categories = [];
 
 		foreach ($pFieldsCollection->getFieldsByModule($module) as $key => $pField) {
+			if (!$this->_showMessageField && $key === 'message') {
```

Comment:
`$_showMessageField` does not belong in this class. Field-specific logic belongs into the classes that build the `FieldsCollection`. This class is supposed to remain stateless.

Reply:
I am having trouble using `removeFieldByModuleAndName` when the module for the Message field is null. I need to check when in Applicant Search Form will remove Message field

Reply:
Doesn't an empty string work for the module then?

Reply:
Yes, the empty module doesn't work with `removeFieldByModuleAndName`

Reply:
@jayay Can you help here?

Reply:
`$pFieldsCollection->removeFieldByModuleAndName('', $fieldName);` should be a perfectly valid invocation (if the module is an empty string, not `null`).

Reply:
But I think `message` shouldn't end up in the `FieldsCollection` in the first place.